Gemfile

```diff
@@ -22,7 +22,7 @@ group :test do
   gem "cucumber", "~> 2.1"
   gem "jekyll_test_plugin"
   gem "jekyll_test_plugin_malicious"
-  gem "codeclimate-test-reporter"
+  gem "codeclimate-test-reporter", "0.6.0"
```

Comment:
@parkr, if you think, going with `"~> 0.6.0"` is the better way for now, I shall definitely move this to a dedicated PR. Otherwise, this will remain here to have the tests run correctly..